wled00/bus_manager.cpp

```diff
@@ -496,32 +512,70 @@ uint32_t BusPwm::getPixelColor(uint16_t pix) const {
 void BusPwm::show() {
   if (!_valid) return;
   unsigned numPins = getPins();
-  unsigned maxBri = (1<<_depth) - 1;
-  // use CIE brightness formula
+  unsigned maxBri = (1<<_depth); // note: not subtraciting 1 ensures full on when set to max (no gpio glitching)
+  unsigned* scaledBri = new unsigned[numPins]; // TODO: could use stack with a fixed size if maximum number of pins is known (or is a #define)
```

Comment:
AFAIK you can use `unsigned scaledBri[numPins];`.
Or use (dreaded) 5 as 5 is the maximum number of pins ever used.

Reply:
`unsigned scaledBri[numPins]` seems to work, the wonders of C++ ...